### ISSUE TITLE
Contact sensor crashes system

### ISSUE THREAD

**Issue by Paul-Reed** [sammachin/node-red-matter-bridge#16]
Using node-RED v4.0.5 - matter-bridge 0.10.1 - Google/Nest speakers

Adding a contact sensor node to my workspace, selecting my default config, and initial state = False.
Full deployment and NR system crashes.
I'm only using one contact sensor node, and it's name is unique.
I am also only using one other matter-bridge node - light dimmer.

```
Starting Bridge
2024-11-30 10:05:35.027 FATAL  Logger               Unhandled error detected: Endpoint ID 2 exists twice
  at InteractionEndpointStructure.verifyAndInitializeStructureElementsFromEndpoint (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:115:13)
  at InteractionEndpointStructure.initializeStructureFromEndpoints (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:79:12)
  at InteractionEndpointStructure.initializeStructureFromEndpoints (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:80:12)
  at InteractionEndpointStructure.initializeFromEndpoint (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:73:10)
  at #updateStructure (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/node/server/TransactionalInteractionServer.js:257:31)
  at new TransactionalInteractionServer (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/node/server/TransactionalInteractionServer.js:118:26)
  at TransactionalInteractionServer.create (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/node/server/TransactionalInteractionServer.js:106:12)
  at processTicksAndRejections (node:internal/process/task_queues:95:5)
  at runNextTicks (node:internal/process/task_queues:64:3)
  at process.processTimers (node:internal/timers:516:9)
30 Nov 10:05:35 - [red] Uncaught Exception:
30 Nov 10:05:35 - [error] Error: Endpoint ID 2 exists twice
    at InteractionEndpointStructure.verifyAndInitializeStructureElementsFromEndpoint (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:115:13)
    at InteractionEndpointStructure.initializeStructureFromEndpoints (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:79:12)
    at InteractionEndpointStructure.initializeStructureFromEndpoints (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:80:12)
    at InteractionEndpointStructure.initializeFromEndpoint (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/protocol/interaction/InteractionEndpointStructure.js:73:10)
    at #updateStructure (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/node/server/TransactionalInteractionServer.js:257:31)
    at new TransactionalInteractionServer (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/node/server/TransactionalInteractionServer.js:118:26)
    at TransactionalInteractionServer.create (/home/me/.node-red/node_modules/@project-chip/matter.js/dist/cjs/node/server/TransactionalInteractionServer.js:106:12)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at runNextTicks (node:internal/process/task_queues:64:3)
    at process.processTimers (node:internal/timers:516:9)
nodered.service: Main process exited, code=exited, status=1/FAILURE
nodered.service: Failed with result 'exit-code'.
nodered.service: Consumed 39.548s CPU time.
```

**Comment by sammachin** [comment]
Did you do a full deploy or just changed nodes?
Currently updating only changed nodes causes these kind of issues, I'm working on a fix for 0.11 at the moment

**Comment by Paul-Reed** [comment]
I was pretty sure that I had done a full deploy, as I was aware of the issue, however, I've just tried it a third time and it did not crash, so I'm now doubting myself...
If so, apologies Sam for wasting your time.
I'll close this for now, and get back to you if it re-occurs.
